example/matchers/consumer/tests/Service/MatchersTest.php

```diff
@@ -113,6 +113,24 @@ public function testGetMatchers(): void
                     [$this->matcher->regex(null, 'car|bike|motorbike')]
                 ),
                 'url' => $this->matcher->url('http://localhost:8080/users/1234/posts/latest', '.*(\\/users\\/\\d+\\/posts\\/latest)$', false),
+                'matchAll' => $this->matcher->matchAll(
+                    ['desktop' => '2000 usd'],
```

Comment:
is this the generator?

should it be enforced to have `atLeast(2)` in length

it gets stored in the body of the pact as

```json
            "matchAll": {
              "desktop": "2000 usd"
            },
```

In our incoming query we send 

`['desktop' => '2000 usd']` (line 220)

Which doesn't match the `atLeast(2)` length restriction.

Reply:
> is this the generator?

No, it's example value.

> should it be enforced to have atLeast(2) in length

Because it's just example value, it doesn't have to. But yes, it make sense to be at least 2 items. I'm just lazy to do so.

> In our incoming query we send `['desktop' => '2000 usd']` (line 220)

It's response from mock server, not request. And mock server just return the example value, doesn't validate at this point.